src/components/NotificationDialog.vue

```diff
@@ -165,6 +165,7 @@ export default {
                 "whapi": "WhatsApp (Whapi)",
                 "gtxmessaging": "GtxMessaging",
                 "Cellsynt": "Cellsynt",
+                "SendGrid": "SendGrid (Email)"
```

Comment:
What do you mean by `(Email)` here?
I don't think this is quite nessesary, right?

```suggestion
                "SendGrid": "SendGrid"
```